doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -145,15 +142,15 @@ Update ACRN Hypervisor Image
 
    .. code-block:: none
 
-      $ make hypervisor BOARD=nuc11tnbi5  SCENARIO=logical_partition RELEASE=0
+      $ make hypervisor BOARD=nuc11tnbi5 SCENARIO=partitioned RELEASE=0
```

Comment:
While we're at it, I'd recommend we remove the `RELEASE` parameter, it does not add any value. Should we want to keep it, let's at least change it to the proper usage which is `RELEASE=[y|n]` these days.
```suggestion
      $ make hypervisor BOARD=nuc11tnbi5 SCENARIO=partitioned
```